wordpress.yaml

```diff
@@ -1,7 +1,136 @@
 ---
-- name: Deploy Wordpress
+- name: Setting up wordpress
   hosts: all
   tasks:
+    - name: Update repositories 
+      become: true
+      apt:
+        #upgrade: yes
+        update_cache: yes
+        cache_valid_time: 86400 #A day
+      
+    - name: Install php dependencies
+      become: true
+      apt:
+        name:
+          - php-cli
+          - php-fpm
+          - php-mysql
+          - php-json
+          - php-opcache
+          - php-mbstring
+          - php-xml
+          - php-gd
+          - php-curl
+        state: present
+     
+    - name: Install nginx
+      become: true
+      apt:
+        name: nginx 
+        state: present
 
-    - name: message to remove
-      command: echo 'Here you gonna put all tasks and roles that gonna be runing on the VM'
+    - name: Open firewall for nginx
+      become: true
+      ufw:
+        rule: allow
+        name: 'Nginx Full'
+    
+    - name: Copy NGINX default settings file
+      become: true
+      copy:
+        src: /vagrant/default
+        dest: /etc/nginx/sites-enabled
+        force: yes
+
+    - name: Install MySQL Server
+      become: true
+      apt:
+        name: 
+          - mysql-server
+          - python3-pymysql  
+        state: present
+    
+    # Set MySQL root Password on Ansible/Ubuntu - https://linuxhint.com/set_mysql_root_password_ansible/ 
+    - name: Relax the MySQL root user
+      shell: sudo mysql -u root -e 'UPDATE mysql.user SET plugin="mysql_native_password"
+       WHERE user="root" AND host="localhost"'
+
+    - name: Flush Privileges
+      shell: sudo mysql -u root -e 'FLUSH PRIVILEGES'
+
+    - name: Create SQL user 
+      mysql_user:
+        login_host: 'localhost'
+        login_user: 'root'
+        login_password: ''
+        name: wp_user
+        password: wp_password
+        priv: "*.*:ALL"
+        state: present
+     
+    - name: Create DB for wordpress
+      mysql_db:
+        login_host: 'localhost'
```

Comment:
what does mean `login_host`?

Reply:
I was trying to follow _verbatim_ the Ansible docs and the login host was included in all examples. It made sense at the time. However, I actually tried the tasks without this and it seems that this property defaults to 'localhost' if not specified, so it's redundant indeed.